lib/Jsrt/ChakraDebug.h

```diff
@@ -22,7 +22,7 @@
 
 #include "ChakraCommon.h"
 
-#if !defined(_WIN32) || !defined(_MSC_VER)
+#if (!defined(_WIN32) || !defined(_MSC_VER)) && !defined(PAL_DEFINES_MSTYPES)
```

Comment:
Since this is public JSRT header, `PAL_...` term here sounds a bit strange to external user. Maybe be more generic? (Expect most of defs here would go away.)